src/samples/OpenChannelMessageThreading.js

```diff
@@ -0,0 +1,684 @@
+import { useState, useRef } from 'react';
+import { v4 as uuid } from 'uuid';
+
+import SendbirdChat, { UserUpdateParams } from '@sendbird/chat';
+import {
+    OpenChannelModule,
+    OpenChannelHandler,
+    OpenChannelCreateParams,
+    OpenChannelUpdateParams
+} from '@sendbird/chat/openChannel';
+
+import {
+    UserMessageUpdateParams,
+    UserMessageCreateParams,
+    MessageListParams,
+    FileMessageCreateParams,
+    MessageRetrievalParams,
+    ThreadedMessageListParams
+} from '@sendbird/chat/message';
+
+import { SENDBIRD_INFO } from '../constants/constants';
+import { timestampToTime } from '../utils/messageUtils';
+
+let sb;
+
+const BasicOpenChannelSample = (props) => {
+
+    const [state, updateState] = useState({
+        currentlyJoinedChannel: null,
+        currentlyUpdatingChannel: null,
+        treadsParentsMessage: {},
```

Comment:
Treads typo.

Would do a search for this typo as I have seen it in a few places.